plugins/modules/vmware_deploy_ovf.py

```diff
@@ -70,6 +70,11 @@
         description:
         - Disk provisioning type.
         type: str
+    enable_hidden_properties:
+        description:
+        - Enable source properties that are marked as ovf:userConfigurable=false
+        default: false
+        version_added: '3.9.0'
```

Comment:
```suggestion
        version_added: '3.11.0'
```

3.9.0 (and meanwhile also 3.10.0) have already been released.